src/core/server/base.js

```diff
@@ -403,17 +403,19 @@ export function runServer({
         let server = baseServer(
           routes, createStore, { appInstanceName: appName });
         if (listen === true) {
-          if (useHttpsForDev && (host === 'example.com')) {
-            const options = {
-              key: fs.readFileSync('bin/local-dev-server-certs/my-server.key.pem'),
-              cert: fs.readFileSync('bin/local-dev-server-certs/my-server.crt.pem'),
-              ca: fs.readFileSync('bin/local-dev-server-certs/my-private-root-ca.crt.pem'),
-              passphrase: '',
-            };
-            server = https.createServer(options, server);
-          } else {
-            log.info(
-              `To use the HTTPS server you must serve the site at example.com (host was "${host}")`);
+          if (useHttpsForDev) {
+            if (host === 'example.com') {
+              const options = {
+                key: fs.readFileSync('bin/local-dev-server-certs/my-server.key.pem'),
```

Comment:
Instead of `my-server`, I'd rename the keys with the host name like `example.com.key.pem` and same for the `crt`, because it is a bit confusing (or name the directory differently maybe).

Reply:
Good catch -- this change would be helpful